examples/DEMO4-digits.md

```diff
@@ -0,0 +1,237 @@
+# DEMO 4: White-box attack of an on-disk
+[[Demo Home]](../demo/README.md)
+## Objective
+Compare white-box and API attacks to digits model
+
+## Threat Model
+- Attacker knowledge: full access to model
+- Security violation: integrity
+
+1. Start the Counterfit CLI.
+    ```
+    $ python examples/terminal/terminal.py 
+   
+                              __            _____ __
+      _________  __  ______  / /____  _____/ __(_) /_
+     / ___/ __ \/ / / / __ \/ __/ _ \/ ___/ /_/ / __/
+    / /__/ /_/ / /_/ / / / / /_/  __/ /  / __/ / /
+    \___/\____/\__,_/_/ /_/\__/\___/_/  /_/ /_/\__/
+
+                    Version: 1.1.0
+    ```
+
+
+2. List available targets
+    ```
+    counterfit> list targets 
+    ┏━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+    ┃ Name            ┃ Model Type ┃ Data Type ┃ Input Shape   ┃ # Samples ┃ Endpoint                                             ┃
+    ┡━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+    │ creditfraud     │ blackbox   │ tabular   │ (30,)         │ 0         │ creditfraud/creditfraud_sklearn_pipeline.pkl         │
+    │ digits_blackbox │ blackbox   │ image     │ (1, 28, 28)   │ 0         │ digits_blackbox/mnist_sklearn_pipeline.pkl           │
+    │ digits_keras    │ keras      │ image     │ (28, 28, 1)   │ 0         │ digits_keras/mnist_model.h5                          │
```

Comment:
Same comment as above, could you update the output by re-running this command with latest code.

Reply:
@dlmgary I'm unable to see the update from `keras` to `blackbox`?

Reply:
Fixed in 658c044191818a37e68c4d6f5d444d551740609b.